src/renderer/components/HomeTab/index.jsx

```diff
@@ -145,9 +146,16 @@ class RecentInvestJobs extends React.PureComponent {
 
     return (
       <Container>
-        <h4>
-          Recent runs:
-        </h4>
+        <div className="mb-1">
+          <h4 className="d-inline-block">
+            Recent runs:
+          </h4>
+          <OpenButton

```

Comment:
> To my eye this would look better with the Open button and corresponding "Try the Open button..." text placed above "Recent runs:". 

Yeah I see your point here. I'm changing this to replace the "Recent runs" text with the "Try the Open..." text when there are no recent runs. 

> I think it could make sense to have it even if there are recent runs to show. Maybe eventually, the "Try the Open button..." text could be part of a getting started/"tour" feature that only shows up the first time you use the app.

I don't really think the "Try the open button" button should always be present because once you've used it once, that text just becomes extra clutter.

Reply:
Thanks, it does look neater leaving out the "Recent runs:" text when there aren't any recent runs.

When there are recent runs, the placement of the Open button directly across from "Recent runs:" makes me think it would open a recent run, not open a datastack. But the tooltip text does make that clear. Maybe that's something to address in a future, general UX issue.

Reply:
In a way it could be thought of as opening a recent run, as it can load from a logfile. Maybe it ought to have a 📂 to indicate it's a "browse" button rather than being associated with the available recent runs. Maybe I'll do that in #113 where I'm adding other emoji.